Add Tutorial on CargoRocket Mapathon (#9)
CargoRocket/website: _posts/2021-04-17-mapping-tutorial.md

## Einführung OSM

*Wenn du bereits bei OSM aktiv bist, kannst du diesen Abschnitt vermutlich überspringen.*

Für das Eintragen der Daten wird zunächst ein OSM-Nutzerkonto benötigt, welcher [hier](https://www.openstreetmap.org/user/new) erstellt werden kann. Für das Bearbeiten existieren verschiedenste Editoren. Für den Einstieg sind folgende am besten geeignet: [ID-Editor](https://learnosm.org/de/beginner/id-editor/) und [StreetComplete](https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete). Für unterwegs auf dem Smartphone existieren auch [Vespucci](https://play.google.com/store/apps/details?id=de.blau.android&hl=de&gl=US) (Android) und [Go Map!!](https://apps.apple.com/de/app/go-map/id592990211) (iOS) um eine Auswahl zu erwähnen.

Eine allgemeine Einführung gibt es bei [LearnOSM](https://learnosm.org/de/), denn dies soll keine Anleitung für OSM selbst sein.

## Tags

> Wir haben verschiedene Faktoren identifiziert, welche wichtig für Lastenrad-GIS-Applikationen sind. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende Tags, welche die Eigenschaften von Wegen und Barrieren beschreiben:

Reviewer: ```suggestion
Wir haben verschiedene Faktoren identifiziert, die die Lastenrad-Tauglichkeit von Straßen ausmachen. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende Tags, welche die Eigenschaften von Wegen und Barrieren beschreiben:
```

GIS ist eigentlich Software wie QGIS oder ArcGIS.

Reviewer 2: Finde die Formulierung auch offener. Zählt eine einfache Karte nicht auch schon als [GIS](https://de.wikipedia.org/wiki/Geoinformationssystem)?

Reviewer: ob es offiziell dazu zählt, bin ich nicht sicher. Aber eine Routing App würde man üblicherweise jetzt nicht als GIS-Software bezeichnen.

* Radwege
  *  [`surface=*`](https://wiki.openstreetmap.org/wiki/DE:Key:surface)
  *  [`smoothness=*`](https://wiki.openstreetmap.org/wiki/DE:Key:smoothness)
  *  [`width=*`](https://wiki.openstreetmap.org/wiki/DE:Key:width)
*  Poller
   *  [`maxwidth:physical=*`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)

> 

Reviewer: Zusätzlich sollten noch nicht in OSM getaggte Barrieren wie Poller, Umlaufgitter und Bordsteine hinzugefügt werden:
- 'barrier' = bollard, cycle_barrier, kerb

Author: Diese Liste von Tags sollte nur Eigenschaften auflisten, die für uns relevant sind nicht die "Haupt"-Tags selbst. Oder würdest du diese auch hinzufügen?

Reviewer: ich hatte das so verstanden, dass hier alle relevanten Tags, die im Mapathon erfasst werden sollen gelistet sind. 
Wir haben jetzt ja zusätzlich zu den ergänzenden Tags noch aufgenommen, dass wir zählen, wieviel neue Barrieren / Bordsteine erfasst werden.
Unten kommen die Tags ja "nur noch" unter 'Beispiele', wo beschrieben ist, wie genau etwas getaggt werden soll.
Insofern würde das für mich schon Sinn machen das hier noch zu erwähnen.

Reviewer: hab gerade gesehen, dass du die Subtags für Bordsteine noch hinzugefügt hast. Dann ist das vielleicht auch verständlich genug so.
Oder einfach noch in Klammern dahinter?
`Poller (barrier=bollard)` & `Umlaufgitter (barrier=cycle_barrier`)

> **Vorsicht**: Es gibt allerdings viele verschiedene Möglichkeiten und entsprechende [Sub-Tags (Namespaces)](https://wiki.openstreetmap.org/wiki/DE:Attribut#Schl.C3.BCssel_und_Werte), die bspw. den Radstreifen neben der Fahrbahn (`cycleway:right=lane`). Diese werden bevorzugt, wenn vorhanden.

## Maßnehmen

Für das ausmessen von Breiten der Wege bzw. Barrieren sollte entweder ein klassisches Maßband oder eine entprechende App (bspw. [Android](https://play.google.com/store/apps/details?id=com.google.tango.measure) bzw. [iOS](https://play.google.com/store/apps/details?id=com.google.tango.measure&hl=de&gl=US)) verwendet werden. Somit ist eine hohe Genauigkeit gewährleistet.

> Falls das Maß nur geschätzt werden kann, könnte [est_width=*](https://wiki.openstreetmap.org/wiki/Key:est_width) verwendet werden. Dies wird allerdings nicht von uns ausgewertet.

Reviewer: Hier wird das zB gelöst über "source:width=estimated" 
https://www.openstreetmap.org/way/26820200#map=18/48.01495/7.85443

Author: Valider punkt! Das füge ich noch hinzu.

Einen abgetrennter, beschildeter Radweg könnte bspw. folgende Tags enthalten:
* `highway=cycleway`
* `surface=asphalt`
* `smoothness=excellent`
* `width=2.0` (in Metern)
* `traffic_sign=DE:237`

### Zweirichtungs-Radweg

<img alt="Zweirichtungsradweg mit exzelletem Untergrund" src="/assets/images/bidirectional-cycleway.jpg" width= "200" class="float left">

Einen abgetrennter, exklusiver Radweg hätte bspw. folgende Tags:
* `highway=cycleway`
* `oneway=no`
* `surface=asphalt`
* `smoothness=excellent`
* `width=3.5`
* `traffic_sign=DE:237`

### Radstreifen

> **Vorsicht**! Das `right` in `cycleway:right` bezieht sich immer auf die Laufrichtung des OSM Weges.
> Und hier bezieht sich `smoothness` auf die Fahrbahn und `cycleway:right:smoothness` auf den Radstreifen.

<img alt="Radstreifen in mäßigem Zustand" src="/assets/images/cycleway:lane.jpg" width= "200" class="float right">

Eine Straße mit Radstreifen, der parallel zur Fahrbahn verläuft, hat möglicherweise folgende Tags:
* `highway=residential`
* `surface=asphalt`
* `cycleway:right=lane`
* `cycleway:right:lane=exclusive`

> * `cycleway:right:smoothness=good`

Reviewer: wollen wir hier nochmal explizit darauf eingehen, ob warum `cycleway:right:surface` nicht extra getaggt ist?
-> man könnte es extra taggen
-> ist aber nicht notwenig, da bei Radweg-Typ "lane" (anders als bei track) davon ausgegangen wird, dass die Oberfläche die gleiche ist wie die der Straße.

So ermittele ich das zumindest aktuell

Author: Yes, ist geändert!

* `cycleway:right:width=1.7`
* `cycleway:right:traffic_sign=DE:237` (nur, wenn beschildert)
* `smoothness=excellent` (bezieht sich auf die Fahrbahn)

### Getrennter Rad-& Fußweg

<img alt="Getrennter, ausgeschilderter Radweg und Fußweg" src="/assets/images/separated-cycleway.jpg" width= "200" class="float left">

Ein ausgewiesener, getrennter Rad- und Fußweg könnte folgende Tags enthalten:
* `highway=path`
* `bicycle=designated`
* `foot=designated`
* `traffic_sign=DE:241` (nur wenn beschildert)
* `segregated=yes`
* `surface=paving_stones`
* `cycleway:surface=*` (nur, falls sich vom Rest des Weges unterscheiden würde)
* `cycleway:smoothness=intermediate`
* `cycleway:width=0.7`

### Bordsteine

Bordsteine können insbesondere für Schwerlast-Lastenräder eine erhebliche Barriere darstellen und sollten deshalb in OpenStreetMap eingetragen werden. Sie werden als Punkt auf Wegen getaggt. In Baden-Württemberg existieren auch von behördlicher Seite keine flächendeckenden Informationen über Bordsteine. 

> 

Reviewer: Sollen wir hier nochmal explizit sagen, dass nur relevante Bordsteine (welche, über die ein Fahrrad auch fahren muss) getaggt werden müssen? Also sprich diese auch nur als Punktgeometrien auf Wegen relevant sind? Man könnte ja auch Linien entlang aller Gehwege ziehen...

<img alt="Bordsteine an Kreuzung an Verkehrsinseln" src="/assets/images/kerb.jpg" width= "300" class="float right">

* [`barrier=kerb`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=kerb)
* [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb)`=raised`
* `height=0.07` (in Metern)

> 

Reviewer: haben wir hier vielleicht noch ein Bild von einem Bordstein "raised"?
und vielleicht können wir noch die Info (laut der Kerb OSM Wiki Seite) dazu schreiben, dass 3cm so ca. die Unterscheidungsgrenze zwischen raised und lowered ist (und dass es theoretisch noch flush für ebenerdige Bordsteine gibt)

Author: Ich hätte dahingehend auf die OSM Wiki Seite verwiesen, aber vielleicht finde ich heute noch eine entsprechende Stelle :)

Und auf Straßen, bei denen der Radweg auf diesen Wegen gemappt ist:

* `cycleway:left:barrier=kerb`
* [`kerb`](https://wiki.openstreetmap.org/wiki/DE:kerb)`=lowered`
* `height=0.04` (in Metern)

### Poller

<img alt="Beispiel für schmale Pollerdurchfahrt vor Brücke" src="/assets/images/bollard.jpg" width= "300" class="float right">

An Pollern ist vor allem eines Interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden.

* [`barrier=bollard`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=bollard)
* [`bollard`](https://wiki.openstreetmap.org/wiki/DE:Key:bollard)`=foldable`
* [`maxwidth:physical`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)`=1.1`
* `bicycle=yes`
* `foot=yes` (wenn für Fußverkehrerlaubt)

### Umlaufgitter

Für Umlaufgitter, oder auch Drängelgitter genannt, existieren derzeit noch keine gut verwendbare Tags um diese sinnvoll zu beschreiben. Nichtsdestotrotz sollten auch diese in OpenStreetMap eingetragen werden. Diese werden als Punkt auf einem Weg getaggt.

> 

Reviewer: Wie wäre es damit:
Für Umlaufgitter, oder auch Drängelgitter genannt, sollte die engste Stelle gemessen und als maxwidth:physical angegeben werden. Dies kann entweder der Abstand zwischen den beiden Gittern sein, als auch der Abstand vom Gitter zu einer möglichen seitlichen Begrenzung (siehe Bilder).
Diese Angabe reicht zwar nur bedingt, um ein Durchkommen für Lastenräder festzustellen, da auch bei ausreichender Breite die Kurve zu eng für ein Durchkommen sein kann, allerdings gibt die Breite zumindest einen gewissen Anhaltspunkt. 
Zusätzlich bietet diese Breitenangabe beispielsweise für Rollstuhlfahrende eine ausreichende, wichtige Information.

Bilder: https://drive.google.com/file/d/1yqbLU6iqJvE568rJVhsK8DYbAeqDTl7S/view?usp=sharing
https://drive.google.com/file/d/1x96lDmIcf2vv-ARAHZs6Hux9YgjfiC9n/view?usp=sharing

Author: Ist geändert.

* `cycleway:smoothness=intermediate`
* `cycleway:width=0.7`

### Bordsteine

Bordsteine können insbesondere für Schwerlast-Lastenräder eine erhebliche Barriere darstellen und sollten deshalb in OpenStreetMap eingetragen werden. Sie werden als Punkt auf Wegen getaggt. Für den Mapathon sind Bordsteine entscheidend, welche als Punkte auf Radwegen liegen. Poller als Linien oder Poller, die nicht auf Radwegen liegen werden von uns nicht weiter beachtet.

In Baden-Württemberg existieren auch von behördlicher Seite keine flächendeckenden Informationen über Bordsteine.

<img alt="Bordsteine an Kreuzung an Verkehrsinseln" src="/assets/images/kerb.jpg" width= "300" class="float right">

* [`barrier=kerb`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=kerb)
* [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb)`=raised`
* `height=0.07` (in Metern)

Und auf Straßen, bei denen der Radweg auf diesen Wegen gemappt ist:

* `cycleway:left:barrier=kerb`
* [`kerb`](https://wiki.openstreetmap.org/wiki/DE:kerb)`=lowered`
* `height=0.04` (in Metern)

### Poller

<img alt="Beispiel für schmale Pollerdurchfahrt vor Brücke" src="/assets/images/bollard.jpg" width= "300" class="float right">

> An Pollern ist vor allem eines Interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle auf dem Weg sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden. Die Breite sollte sich auf den (in der Regel asphalitierten) Bereich des Weges beziehen. Falls die physikalische Begrenzung nicht durch Zäune, Hecken oder andere Barrieren hergestellt ist, sondern bspw. ein Durchkommen über einen Grünstreifen möglich ist, schlagen wir vor zusätzlich den Tag `maxwidth:physical:definite=no` vor. Dieser soll anzeigen, dass ein überwinden der Barriere durch Ausholen über jenen Grünstreifen möglich ist. Umgekehrt würde `maxwidth:physical:definite=yes` ein Durchkommen als *definitiv* unmöglich angezeigt werden.

Reviewer: ```suggestion
An Pollern ist vor allem eines Interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle auf dem Weg sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden. Die Breite sollte sich auf den (in der Regel asphalitierten) Bereich des Weges beziehen. Falls die physikalische Begrenzung nicht durch Zäune, Hecken oder andere Barrieren hergestellt ist, sondern bspw. ein Durchkommen über einen Grünstreifen möglich ist, schlagen wir vor zusätzlich den Tag `maxwidth:physical:definite=no` vor. Dieser soll anzeigen, dass ein überwinden der Barriere durch Ausholen über jenen Grünstreifen auch mit breiteren Fahrzeugen als durch `maxwidth:physical` angegeben, möglich ist. Umgekehrt würde `maxwidth:physical:definite=yes` ein Durchkommen mit breiteren Fahrzeugen als *definitiv* unmöglich angezeigen.
```

Reviewer: ja, ich glaube das ist eine gute  Lösung mit dem Tag :)

Jeder aktualisierte oder hinzugefügte Tag, Punkt oder Weg hilft CargoRocket und allen anderen der OSM-Community! Ziel ist es möglichst gute, akkurate und nachvollziehbare Daten einzutragen. Ganz konkret kann das folgendermaßen erfolgen:

Durch Hinzufügen & Aktualisieren...
* von Wegen (bspw. ein nicht eingetragener Radweg)
* von Punkten (bspw. ein neuer Poller)
* von Tags (bspw. von Durchfahrtsbreiten an Pollern)

## Tags

Wir haben verschiedene Faktoren identifiziert, die die Lastenrad-Tauglichkeit von Straßen ausmachen. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende Tags, welche die Eigenschaften von Wegen und Barrieren beschreiben:

* Radwege
  *  [`surface=*`](https://wiki.openstreetmap.org/wiki/DE:Key:surface)
  *  [`smoothness=*`](https://wiki.openstreetmap.org/wiki/DE:Key:smoothness)
  *  [`width=*`](https://wiki.openstreetmap.org/wiki/DE:Key:width)
* Poller (`barrier=bollard`) & Umlaufgitter (`barrier=cycle_barrier`)
  * [`maxwidth:physical=*`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)
* Bordstein (`barrier=kerb`)
  *  [`height`](https://wiki.openstreetmap.org/wiki/DE:Key:height)

>   *  [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb) [derzeit nur auf Englisch verfügbar]

Reviewer: ```suggestion
  *  [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb) [OSM Wiki derzeit nur auf Englisch verfügbar]
```

In Baden-Württemberg existieren auch von behördlicher Seite keine flächendeckenden Informationen über Bordsteine.

<div class="flex c">
  <figure role="group">
    <img alt="Raised kerb" src="/assets/images/kerb_raised.jpg" width="200">
    <figcaption>
      <code>raised</code>
    </figcaption>
  </figure>
    <figure role="group">
    <img alt="lowered kerb" src="/assets/images/kerb_lowered.jpg" width= "200">
    <figcaption>
      <code>lowered</code>
    </figcaption>
  </figure>
  <figure role="group">
    <img alt="Flushed kerb" src="/assets/images/kerb_flushed.jpg" width="200">
    <figcaption>
      <code>flushed</code>
    </figcaption>
  </figure>
</div>

* [`barrier=kerb`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=kerb)
* [`kerb`](https://wiki.openstreetmap.org/wiki/Key:kerb)`=raised`
* `height=0.07` (in Metern)

> Und auf Straßen, bei denen der Radweg auf diesen Wegen gemappt ist:

Reviewer: ```suggestion
Führt der Radweg über einen Bordstein, ist jedoch bspw. als `cycleway:right=track` nur als Tag einer Straße vorhanden, kann hier nicht ein Bordstein hinzugefügt werden, der auf der Straße liegt. Das würde sonst bedeuten, dass auch die Autos von dieser Barriere betroffen wären. Daher schlagen wir hier dieses Tagging vor:
```

* `cycleway:left:barrier=kerb`

> * [`kerb`](https://wiki.openstreetmap.org/wiki/DE:kerb)`=lowered`

Reviewer: Wie ist das hier in OSM üblich? So wie von dir beschrieben, oder müsste das so sein?

* `cycleway:left:barrier:kerb=lowered`
* `cycleway:left:barrier:kerb:height=0.04`

Author: Es gibt scheinbar weder noch. Ich hätte ohne `barrier` vorgeschlagen, da sonst zu lange.

* `height=0.04` (in Metern)

### Poller

<img alt="Beispiel für schmale Pollerdurchfahrt vor Brücke" src="/assets/images/bollard.jpg" width= "300" class="float right">

> An Pollern ist vor allem eines Interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle auf dem Weg sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden. Die Breite sollte sich auf den (in der Regel asphalitierten) Bereich des Weges beziehen. Falls die physikalische Begrenzung nicht durch Zäune, Hecken oder andere Barrieren hergestellt ist, sondern bspw. ein Durchkommen über einen Grünstreifen möglich ist, schlagen wir vor zusätzlich den Tag `maxwidth:physical:definite=no` vor. Dieser soll anzeigen, dass ein überwinden der Barriere durch Ausholen über jenen Grünstreifen auch mit breiteren Fahrzeugen als durch `maxwidth:physical` angegeben, möglich ist. Umgekehrt würde `maxwidth:physical:definite=yes` ein Durchkommen mit breiteren Fahrzeugen als *definitiv* unmöglich anzeigen.

Reviewer: ```suggestion
An Pollern ist vor allem eines interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle auf dem Weg sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden. Die Breite sollte sich auf den (in der Regel asphalitierten) Bereich des Weges beziehen. Falls die physikalische Begrenzung nicht durch Zäune, Hecken oder andere Barrieren hergestellt ist, sondern bspw. ein Durchkommen über einen Grünstreifen möglich ist, schlagen wir vor zusätzlich den Tag `maxwidth:physical:definite=no` vor. Dieser soll anzeigen, dass ein überwinden der Barriere durch Ausholen über jenen Grünstreifen auch mit breiteren Fahrzeugen als durch `maxwidth:physical` angegeben, möglich ist. Umgekehrt würde `maxwidth:physical:definite=yes` ein Durchkommen mit breiteren Fahrzeugen als *definitiv* unmöglich anzeigen.
```

* [`barrier=bollard`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=bollard)
* [`bollard`](https://wiki.openstreetmap.org/wiki/DE:Key:bollard)`=foldable`

> * [`maxwidth:physical`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)`=1.1` (F)

Reviewer: was heißt (F)?

Author: Wer kennts nicht: (F) :D
Hat da eig nichts zu suchen  :)

* `bicycle=yes`

> * `foot=yes` (wenn für Fußverkehrerlaubt)

Reviewer: ```suggestion
* `foot=yes` (wenn für Fußverkehr erlaubt)
```

Eine allgemeine Einführung gibt es bei [LearnOSM](https://learnosm.org/de/), denn dies soll keine Anleitung für OSM sein.

## Tags
Wir haben verschiedene Faktoren identifiziert, welche wichtig für Lastenrad-GIS-Applikationen sind. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende generellen Tags:

* Radwege
  *  [`surface=*`](https://wiki.openstreetmap.org/wiki/DE:Key:surface)
  *  [`smoothness=*`](https://wiki.openstreetmap.org/wiki/DE:Key:smoothness)
  *  [`width=*`](https://wiki.openstreetmap.org/wiki/DE:Key:width)
*  Poller
   *  [`maxwidth:physical=*`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)

> Vorsicht: Es gibt allerdings viele verschiedene Möglichkeiten und entsprechende [Sub-Tags (Namespaces)](https://wiki.openstreetmap.org/wiki/DE:Attribut#Schl.C3.BCssel_und_Werte), die bspw. den Radstreifen neben der Fahrbahn (`cycleway:right=lane`). Diese werden bevorzugt, wenn vorhanden.

## Beispiel Szenarien

> ### Eigener Radweg

Reviewer: Vielleicht das als Beispielbild? https://drive.google.com/file/d/1kb58eejB77psl5ldMcQS9XhT_ZPq9SKb/view?usp=sharing

Author: Mhm vorläufig wäre das passend! Wollte am besten noch eins mit Zeichen 237 (Radweg) 
![grafik](https://user-images.githubusercontent.com/6567206/114668484-17843300-9d01-11eb-8551-e45a8ec99494.png)

Aber dann integriere ich das erstmal so!

Reviewer: ah I see. Ich schau mal ob ich an einem vorbei fahre die Tage. Sonst ist "designated" oder nicht für uns ja eigentlicih gar nicht so relevant, oder?

Reviewer: wie wärs damit? https://drive.google.com/file/d/17ojUjZVwPrCuM7dbI9KgJ6JkbXQKOlWb/view?usp=sharing

Author: Danke, optimal! Was genau meinst du mit "designated"? highway=path bicycle=designated?

Reviewer: Ja genau. Ich hab das so verstanden, dass "designated" bedeutet, ob es (u.a. durch das blaue Schild) rechtsverbindlich ist, oder nur "empfholen" aber man als Radfahrer*in auch auf der Straße fahren darf. 
Das berücksichtigen wir (zumindest bisher) ja nicht

Einen abgetrennter, exklusiver Radweg hätte bspw. folgende Tags:
* `highway=cycleway`
* `surface=asphalt`
* `smoothness=excellent`
* `width=3.0`

### Radstreifen

> <img alt="Radstreifen in mäßigem Zustand" src="/assets/images/cycleway:lane.jpg" width= "200" style="float: right;">

Reviewer: Da stimmt noch was nicht mit der Darstellung 

<img width="982" alt="Bildschirmfoto 2021-04-14 um 08 41 31" src="https://user-images.githubusercontent.com/18367515/114668098-9d53ae80-9d00-11eb-8dc3-ceea4a8adbac.png">

Author: Yes auf mobile habe ich gesehen auch nocht nicht :/

Author: Mobile style ist erledigt :)

Eine Straße mit Radstreifen, der parallel zur Fahrbahn verläuft, hat möglicherweise folgende Tags:
* `highway=residential`
* `surface=asphalt`
* `cycleway:right=lane`
* `cycleway:right:lane=exclusive`
* `cycleway:right:smoothness=intermediate`
* `cycleway:right:width=1.7`
* `smoothness=excellent`

> Achtung! Das `right` in `cycleway:right` bezieht sich immer auf die Laufrichtung des OSM Weges.
> Und hier bezieht sich `smoothness` auf die Fahrbahn und `cycleway:right:smoothness` auf den Radstreifen.

### Gemeinsamer Rad-& Fußweg

> <img alt="Getrennter, ausgeschilderter Radweg und Fußweg" src="/assets/images/cycleway:lane.jpg" width= "200" style="float: left; margin-right: 2rem;">

Reviewer: Das BIld hier ist nicht was du meinst oder?
Alternativ vielleicht das? https://drive.google.com/file/d/1cOpTtpxYTPjCqZHxplXDpgg1PAcr6vxR/view?usp=sharing

> Ein ausgewiesener, getrennter Rad- und Fußweg könnte folgende Tags enthalten:

Reviewer: je nach Bild müsste das wahrscheinlich mit den Tags angepasst werden. Wäre aber nciht meistens eher der Fall: "highway=footway"?
Und "width" müsste nochmal zusätzlich über "cycleway:width" ausgewiesen werden, oder? nur der width parameter bringt uns hier ja nichts?

Author: Ja genau, habs geändert :)

* `highway=path`
* `bicycle=designated`
* `foot=designated`
* `traffic_sign=DE:241`
* `segregated=yes`
* `cycleway:surface=asphalt`
* `footway:surface=cobblestone`
* `cycleway:smoothness=excellent`
* `width=3.5`

### Poller

> <img alt="Beispiel für schmale Pollerdurchfahrt vor Brücke" src="/assets/images/bollard_small.jpg" width= "300" style="float: right;">

Reviewer: Vielleicht zur Verdeutlichung sowas: https://drive.google.com/file/d/18UVMa-NV0kdzdtKgzocrNQw0x-DDYwSz/view?usp=sharing ?

Author: Das ist suuper!

An Pollern ist vor allem eines Interessant: Das Durchkommen zwischen mehreren Pollern/Steinen oder Poller und Wegerand. Die breitest mögliche Stelle sollte dann gemessen werden und mit maxwidth:physical in Metern (ohne Einheit anhängen) getaggt werden. Hierfür sollte entweder ein klassisches Maßband oder eine entprechende App (bspw. [Android](https://play.google.com/store/apps/details?id=com.google.tango.measure) bzw. [iOS](https://play.google.com/store/apps/details?id=com.google.tango.measure&hl=de&gl=US)) verwendet werden.

Der Poller als Punkt auf einem Weg hätte bspw. folgende Werte:
* [`barrier=bollard`](https://wiki.openstreetmap.org/wiki/DE:Tag:barrier=bollard)
* [`bollard`](https://wiki.openstreetmap.org/wiki/DE:Key:bollard)`=foldable`
* [`maxwidth:physical`](https://wiki.openstreetmap.org/wiki/Key:maxwidth:physical)`=1.1`
* `bicycle=yes`
* `foot=yes`

> 

Reviewer: brauchen wir hier noch ein Beispiel für cycle_barrier?

Author: Oh, du hast block und cycle barrier auch integriert! Ich dachte es sei nur bollard. Ja, das füge ich noch hinzu.

## Weiterführende Informationen

* Weitere, für Lastenräder wichtige Tags können auf der [Cargobike OSM Wiki Seite](https://wiki.openstreetmap.org/wiki/DE:Cargobike) nachgelesen werden. 
* Einen guten Überblick über umfassendes, generelles taggen von Radinfrastruktur in OSM gibt es von [Verkehrswende Berlin](https://wiki.openstreetmap.org/wiki/Berlin/Verkehrswende/Radwege).

> * Die Allgemeinen Kartierungsempfehlungen sind ebenfalls im OSM Wiki auf der Seite [Radverkehrsanla   gen kartieren](https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren) zu finden.

Reviewer: ```suggestion
* Die Allgemeinen Kartierungsempfehlungen sind ebenfalls im OSM Wiki auf der Seite [Radverkehrsanlagen kartieren](https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren) zu finden.
```
